pkg/opavalidation/opavalidation.go

```diff
@@ -47,8 +47,8 @@ func Run(regoFileName, objectFileName string, insightsInfo fwrego.InsightsInfo,
 	if err != nil {
 		return actionItems, err
 	}
-	if len(actionItems) != 1 {
```

Comment:
@rbren I was also returning an error when there were 0 actionItems, since the goal of validation is for the policy to return an actionItem, and for that actionItem to be valid.

With this change, a policy that returns 0 actionItems validates successfully, including bulk validation (Which uses the same logic).

Reply:
Sorry totally forgot about this PR!

I think it's important to be able to test _both_:
* this policy creates an action item for problematic resources
* this policy does not create an action item for good resources

My eventual goal was to use the filename or a flag to give the CLI a hint as to whether or not it expects an action item.

Reply:
> My eventual goal was to use the filename or a flag to give the CLI a hint as to whether or not it expects an action item.

@rbren  What would you like to do for the 0 actionItems case until your above eventual goal is implemented?

Our current CLI -> OPA docs state, in the validation section under bulk validation: All Policies must generate a single Insights Action Item as output to be considered valid.

Reply:
I think we leave this PR open until we can handle that (or close it and come back to it later)